neuralmonkey/encoders/cnn_encoder.py

```diff
@@ -299,15 +299,24 @@ def max_pooling(
             "Specification of a max-pooling layer (number {} in config) "
             'needs to have 3 members: "M", pool size, stride, padding, '
             "was {}").format(layer_num, specification))
-    pool_size, stride, pad = specification[1:]
+    pool_type, pool_size, stride, pad = specification
+
+    if pool_type == "M":
+        pool_fn = tf.layers.max_pooling2d
+    elif pool_type == "A":
+        pool_fn = tf.layers.average_pooling2d
```

Comment:
udělal bych Dict[str, Callable] a tady házel value error kdyby `not in pool_fn_map`, a pak bych volal `pool_fn_map[pool_type]`, ale netrvám na tom, abys to měnil, jen jsem se chtěl podělit o svuj coding style..

Reply:
Jsou dva a víc jich nebude.